src/Commands/core/CacheWarmCommands.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drush\Commands\core;
+
+use Drupal\Core\PreWarm\CachePreWarmerInterface;
+use Drush\Attributes as CLI;
+use Drush\Commands\DrushCommands;
+use Psr\Container\ContainerInterface;
+
+final class CacheWarmCommands extends DrushCommands
+{
+    const WARM = 'cache:warm';
+
+    public function __construct(
+        private CachePreWarmerInterface $warmer,
+    ) {
+        parent::__construct();
+    }
+
+    public static function create(ContainerInterface $container)
+    {
+        if ($container->has('cache_prewarmer')) {
+            return new self($container->get('cache_prewarmer'));
+        }
+        // Do nothing. Command never gets added to the Application.
+    }
+
+    /**
+     * Pre-warm all caches.
+     */
+    #[CLI\Command(name: self::WARM, aliases: ['warm', 'cache-warm'])]
+    #[CLI\Version(version: '13.5')]
+    public function warm(): void
+    {
+        $this->warmer->preWarmAllCaches();
```

Comment:
We could possibly go a step further, and get the prewarmable services from the container, then start a new process to prewarm each one in parallel. There is not a nice API for this so could maybe be a follow-up.